core/node/node_framework/src/implementations/layers/metadata_calculator.rs

```diff
@@ -25,11 +32,23 @@ use crate::{
 /// - Adds `tree_health_check` to the `ResourceCollection<HealthCheckResource>`.
 /// - Adds `metadata_calculator` to the node.
```

Comment:
worth to say that it adds TreeApiTask as well?

Reply:
Doc-comments still will be tackled separately